src/install/config/dotnet.yaml

```diff
@@ -217,14 +217,28 @@ steps:
           - optionType: deployment
             options:
               - value: 'awsLambda'
-  - filePath: 'src/install/dotnet/installation/azure-web-apps.mdx'
+  - optionType: deployment 
+    label: ''
+    placeholder: "How would you like to enable .NET?"
+    options:
+      - value: 'azuresiteextension'
+        displayName: 'Use the Azure Site Extension (Windows Only)'
+      - value: 'azurenugetframework'
+        displayName: 'Use NuGet for .NET Framework'
+      - value: 'azureframeworkcore'
+        displayName: 'Use NuGet for .NET Core'
+      - value: 'azurelocalconfig'
+        displayName: 'Place Local newrelic.config File'        
+      - value: 'azureappconfig'
+        displayName: 'Use Application Config File'
+        - value: 'azureWebApps'
+  - filePath: 'src/install/dotnet/dockerOptions.mdx'
     overrides:
-      - isConditionalStep: true
-        selectedOptions:
-          - optionType: deployment
-            options:
-              - value: 'azureWebApps'
-  - filePath: 'src/install/dotnet/installation/nuget.mdx'
+      selectedOptions:
+        - optionType: deployment
+          options:
+            - value: 'azureappinsights'
```

Comment:
I'm not seeing `azureappinsights` defined anywhere as a value. Your values here should be anything defined under the deployment `optionType` (`azuresiteextension`, `azureframeworkcore`, etc)